service/src/main/java/com/commercetools/pspadapter/payone/domain/payone/model/common/PayoneResponseFields.java

```diff
@@ -5,13 +5,20 @@
     // most of requests have it
     public static final String STATUS = "status";
 
+    public static final String REDIRECT_URL = "redirecturl";
+
     // in case of error response
     public static final String ERROR_CODE = "errorcode";
     public static final String CUSTOMER_MESSAGE = "customermessage";
     public static final String ERROR_MESSAGE = "errormessage";
 
```

Comment:
just use Camelcase for the parameter values

Reply:
it's Payone response variable name - not my choice.